src/brasil/gov/vlibrasvideo/utils.py

```diff
@@ -149,58 +153,52 @@ def repost_news(context, event=None):
         timeout=REQUEST_TIMEOUT
     )
     logger.info('PUT - {0}'.format(params))
-    response = requests.put(**params)
-    logger.info(u'{0}: {1} - {2}'.format(
-        context.absolute_url(), response.status_code, response.reason
-    ))
-    return (response.status_code, response.reason)
+    try:
+        response = requests.put(**params)
+    except RequestException as e:  # skip on timeouts and other errors
+        logger.error(u'{0}: {1}'.format(
+            context.absolute_url(), e.message
+        ))
+        return False
+    if response.status_code != 200:
+        logger.error(u'{0}: {1} - {2}'.format(
+            context.absolute_url(), response.status_code, response.reason
+        ))
+        return False
+    return True
 
 
 def get_video_url(context):
     """Get a video url from VLibrasVideo.
     :param context: [required] Content object
     :type context: content object
-    :returns: Status code and youtube url or error description
-    :rtype: tuple
+    :returns: youtube url if available
+    :rtype: string or None
     """
-    enabled_content_types = _get_registry('enabled_content_types', [])
-    if getattr(context, 'portal_type', '') not in enabled_content_types:
-        status_code = -3
-        reason = u'VLIBRAS VIDEO NOT ENABLED FOR THIS CONTENT TYPE'
-        logger.info(u'{0}: {1} - {2}'.format(
-            context.absolute_url(), status_code, reason
-        ))
-        return (status_code, reason)
     token = _get_registry('vlibrasvideo_token')
-    if not token:
-        status_code = -2
-        reason = u'NEED TO INFORM SECRET TOKEN'
-        logger.info(u'{0}: {1} - {2}'.format(
-            context.absolute_url(), status_code, reason
-        ))
-        return (status_code, reason)
-    if _get_state(context) != 'published':
-        status_code = -1
-        reason = u'UNPUBLISHED CONTENT'
-        return (status_code, reason)
+    if not _validate(context, token):
+        return
     headers = {'Authentication-Token': token}
     params = dict(
         url=VIDEO_URL.format(context.UID()),
         headers=headers,
         timeout=REQUEST_TIMEOUT
     )
     logger.info('GET - {0}'.format(params))
-    response = requests.get(**params)
+    try:
+        response = requests.get(**params)
+    except RequestException as e:  # skip on timeouts and other errors
+        logger.error(u'{0}: {1}'.format(
+            context.absolute_url(), e.message
```

Comment:
the URL of the remote server are always fixed, but the context where we got the exception is not. I think just add a text saying that it is a GET is sufficient to know what is going on here.